### PR TITLE
add cincinnati ci

### PR DESCRIPTION
**Pull request by steveej**
Please help me evaluate if this PR does the following things:

- [ ] enable tide merging using `/lgtm` or `/approve`
- [ ] enable PR job
- [ ] enable master job

Depends on https://github.com/openshift/cincinnati/pull/11.
I've verified that the master job works by locally running:

```sh
	ci-operator \
		--config ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml \
		--git-ref steveej-forks/openshift-cincinnati@test-fetch-releases-against-quay  \
		--namespace steveej-cincinnati-ci
```

### CODE REVIEW

**Comment by steveeJ** [comment]
> The following test **failed**, say `/retest` to rerun them all:
> Test name 	Commit 	Details 	Rerun command
> ci/prow/generated-config 	[05022b8](https://github.com/openshift/release/commit/05022b82999feb7e2032acfba08f7305e67c9d99) 	[link](https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_release/2452/pull-ci-openshift-release-master-generated-config/1799) 	`/test generated-config`

Dang, looks like symlinks to other configs are not supported

**Comment by steveeJ** [comment]
> Dang, looks like symlinks to other configs are not supported

I take that back, the symlink was not the issue and it works with it too now.

@crawford can you PTAL at the OWNERS* in this PR?

File: ci-operator/config/openshift/cincinnati/openshift-cincinnati-master-presubmit.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+base_images:
```
Comment (smarterclayton) [inline ci-operator/config/openshift/cincinnati/openshift-cincinnati-master-presubmit.yaml]: This file shouldn't exist.

Comment (steveeJ) [reply ci-operator/config/openshift/cincinnati/openshift-cincinnati-master-presubmit.yaml]: Isn't it used for PRs?

Comment (smarterclayton) [reply ci-operator/config/openshift/cincinnati/openshift-cincinnati-master-presubmit.yaml]: no, PRs and merges are both handled from openhsift-cincannit-master.  The presubmits jobs are generated from the single config.  We have one file per repo per branch.

Comment (steveeJ) [reply ci-operator/config/openshift/cincinnati/openshift-cincinnati-master-presubmit.yaml]: Alright, removed

File: ci-operator/config/openshift/cincinnati/OWNERS [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+# This file just uses aliases defined in OWNERS_ALIASES.
+
+approvers:
+  - installer-approvers
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/cincinnati/OWNERS]: This seems weird

Comment (steveeJ) [reply ci-operator/config/openshift/cincinnati/OWNERS]: Thanks for catching this, see #2454

**Pull request merged**